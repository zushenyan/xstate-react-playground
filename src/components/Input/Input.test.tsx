import initStoryshots from '@storybook/addon-storyshots';
import 'jest-styled-components';

initStoryshots({ storyKindRegex: /^Input$/ });
