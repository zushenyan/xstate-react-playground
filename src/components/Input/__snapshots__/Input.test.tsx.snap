// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Input Initial Props 1`] = `
.c0 {
  font-size: 20px;
  box-sizing: border-box;
  border: 1px solid black;
  padding: 5px;
}

<input
  className="c0"
  placeholder="hello world"
/>
`;
